packages/terra-date-picker/src/react-datepicker/stylesheets/variables.scss

```diff
@@ -2,7 +2,8 @@ $datepicker-background-color: #f0f0f0 !default;
 $datepicker-border-color: #aeaeae !default;
 $datepicker-highlighted-color: #3dcc4a !default;
 $datepicker-muted-color: #ccc !default;
-$datepicker-selected-color: #216ba5 !default;
+$datepicker-selected-color: #0079be !default;
+$datepicker-hover-color: #0065a3 !default;
```

Comment:
Can we rename `$datepicker-hover-color` to `$datepicker-selected-hover-color` to better represent that this is the hover colored used only when the day is selected.

Reply:
Changed in eee3598